clients/java/src/main/java/io/camunda/zeebe/client/api/command/UpdateTimeoutJobCommandStep1.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.zeebe.client.api.command;
+
+import io.camunda.zeebe.client.api.response.UpdateTimeoutJobResponse;
+
+public interface UpdateTimeoutJobCommandStep1 {
+
+  /**
+   * Set the timeout of this job.
+   *
+   * @param timout the timeout of this job
+   * @return the builder for this command. Call {@link #send()} to complete the command and send it
+   *     to the broker.
+   */
```

Comment:
💭 It might be interesting for users to note any specifics here, e.g. does this add to the existing timeout or replace it, if it replaces it, does it start from now or from when the related command is processed?

Reply:
Changes are ready for review.